sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b3"
```

Comment:
Could we add an entry to the changelog for this new version, listing the updated features/breaking changes?